src/Ot2Rec/main.py

```diff
@@ -1361,3 +1290,218 @@ def run_rlf_deconv():
+    """
+
+    project_name = get_proj_name()
+
+    mc2_yaml = project_name + '_mc2.yaml'
+    recon_yaml = project_name + '_recon.yaml'
+
+    # Create Logger object
+    logger = logMod.Logger()
+
+    if os.path.isfile(mc2_yaml):
+        mc2_config = prmMod.read_yaml(project_name=project_name,
+                                      filename=mc2_yaml)
+        mc2_path = mc2_config.params['System']['output_path']
+        if os.path.isdir(mc2_path):
+            logger(f"Deleting {mc2_path} folder and its contents...")
+            cmd = ['rm', '-rf', mc2_path]
+            del_mc2 = subprocess.run(cmd,
+                                     stdout=subprocess.PIPE,
+                                     stderr=subprocess.STDOUT)
+
+    if os.path.isfile(recon_yaml):
+        recon_config = prmMod.read_yaml(project_name=project_name,
+                                        filename=recon_yaml)
+        recon_path = recon_config.params['System']['output_path']
+        if os.path.isdir(recon_path):
+            logger(f"Deleting intermediary IMOD files...")
+            files = glob(recon_path + 'stack*/*.*~') + \
+                glob(recon_path + 'stack*/*_full_rec.*')
+            cmd = ['rm', *files]
+            del_recon = subprocess.run(cmd,
+                                       stdout=subprocess.PIPE,
+                                       stderr=subprocess.STDOUT)
+
+
+def run_all():
+    """
+    Method to run all four processes in one go using default settings.
+    """
+
+    logger = logMod.Logger()
+
+    # Collect raw images and produce master metadata
+    logger("Collecting raw images...")
+    get_master_metadata()
+
+    # Motion correction
+    logger("Motion correction in progress...")
+    create_mc2_yaml()
+    run_mc2()
+
+    # CTF estimation
+    logger("CTF estimation in progress...")
+    create_ctffind_yaml()
+    run_ctffind()
+
+    # Alignment
+    logger("Alignment in progress...")
+    create_align_yaml()
+    run_align()
+
+    # Reconstruction
+    logger("Reconstruction in progress...")
+    create_recon_yaml()
+    run_recon()
+
+
+def update_aretomo_yaml(args, kwargs):
+    """ Placeholder 
+    Method to update yaml file for AreTomo
+
+    Args:
+    args (Namespace) :: Namespace containing user inputs
+    kwargs (list) :: List of extra inputs, used for extra AreTomo arguments
+                     beyond those implemented here
+    """
+    # Read in YAML, set mundane things
+    rootname    = args.project_name if args.rootname is None else args.rootname
+    suffix      = args.suffix
+
+    aretomo_yaml_name = args.project_name + "_aretomo.yaml"
+    aretomo_params = prmMod.read_yaml(
+        project_name=args.project_name,
+        filename=aretomo_yaml_name
+    )
+
+    # Add optional kwargs
+    for param in kwargs:
+        aretomo_params.params["AreTomo_kwargs"][param] = vars(args).get(param)
+
+
+    # set process list, InMrc, OutMrc, AngFile
```

Comment:
this is where the pre-processing outputs would be required:
- `InMrc` is the input mrc file: either motion-corrected or aligned single .mrc stack
- `OutMrc` is the filename to write the output mrc file
- `AngFile` is the raw or aligned tilt angles data.